configure.ac

```diff
@@ -72,6 +72,9 @@ AM_PROG_AR
                     # it out for now
 LT_INIT
 
+# Initialize pkg-config since we are installing a .pc file
+PKG_INSTALLDIR
```

Comment:
good

Reply:
Unfortunately the pkg-config 0.26 in Ubuntu 14.04 (used by your Travis instance) doesn't support this.  It was introduced in 0.27 released in 2012.